tests/gcp/test_optimizers.py

```diff
@@ -100,3 +100,33 @@ def test_lbfgsb(generate_problem):
     assert isinstance(info, dict)
     assert not model.isequal(result)
     assert (model.full() - dense_data).norm() > (result.full() - dense_data).norm()
+
+
+def test_lbfgsb_callback(generate_problem):
+    dense_data, model, sampler = generate_problem
+
+    # Test default callback
+    maxiter = 2
+    solver = LBFGSB(maxiter=maxiter)
+    _, info = solver.solve(model, dense_data, gaussian, gaussian_grad)
+    assert not "callback" in info["callback"].keys()
+    assert info["callback"]["time_trace"].shape == (maxiter,)
```

Comment:
Can we update this to ensure the timings are greater than zero? When looking at your typing bug I accidentally turned off the timing portion but the member in the Monitor had the shape set correctly (but always returned zero)

Reply:
Yes. That's the request. Right now if ONLY the constructor for the monitor existed this test would still pass.

Reply:
Ah okay, that makes sense.